### ISSUE TITLE
arkserver status shows offline, players continue to play and rcon connects

### ISSUE THREAD

**Issue by CharlieD573** [arkmanager/ark-server-tools#979]
I seem to be having a strange issue that sounds similar to one that was resolved months ago.

When I run arkmanager status, some or all of the instances of my server cluster show 'Server online: No'

I panicked a little bit at this, restarted the cluster, restarted the server... the issue keeps randomly cropping up.  This morning, I was connected via the Rcon while a few of my players were happily running around, while the 'status' command informed me that the server was once again not online.  My players and the Rcon disagree.

I'm running version 1.6.42

**Comment by spacefiddle** [comment]
Old - If anyone else comes by and sees this...

This is a transient Steam issue. The output of status shows Running, Listening, and Online, but doesn't really explain the difference.

Running: should be obvious really
Listening: ports are open and available
Online: this is the part that depends on Steam. If Steam doesn't see your server for any reason - as in, the server is running, ports are open, and players can connect, but the steam browser is having a brain fart of some kind - the Online will show No even if Running and Listening are both Yes and players are having no issues.

TL;DR notabug
